data/json/effects.json

```diff
@@ -1034,7 +1034,7 @@
         "type": "effect_type",
         "id": "flu",
         "name": ["Influenza"],
-        "desc": ["Symptoms alleviated by medication (cough syrup)."],
+        "desc": ["You have a flu.  Symptoms can be alleviated by medication (cough syrup)."],
```

Comment:
I'd recommend `"You have contracted the 'flu'.`, or instead of slang, `"You have contracted influenza.`

Reply:
Same here, do you even say in a casual conversation "I have contracted the 'flu'." ? There is no need to go all stiff on those messages imho.